1302.py

```diff
@@ -0,0 +1,28 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def deepestLeavesSum(self, root: TreeNode) -> int:
+        if not root:
+            return 0
+
+        queue = collections.deque([root])
+
+        while queue:
+            level = []
+            level_length = len(queue)
+
+            for _ in range(level_length):
+                node = queue.popleft()
+                level.append(node.val)
+
+                if node.left:
+                    queue.append(node.left)
+
+                if node.right:
+                    queue.append(node.right)
+
+        return sum(level)
```

Comment:
I'm concerned about the scope of the `level` variable. It works in Python (and certainly not in Java) but I don't think it's a good practice. The usage of `level` should be limited within the `while` loop. 

Similarly, the following works but I don't think it's a good practice:
```python
for i in range(10):
  # do something here
  # ...
  # ...

print (i) # prints 9